pkg/azureclients/azure_client_config.go

```diff
@@ -61,6 +63,15 @@ type RateLimitConfig struct {
 	CloudProviderRateLimitBucketWrite int `json:"cloudProviderRateLimitBucketWrite,omitempty" yaml:"cloudProviderRateLimitBucketWrite,omitempty"`
 }
 
+func (cfg *ClientConfig) CheckARG(enabledARGClients map[string]bool, client string) *ClientConfig {
+	newClientConfig := *cfg
+	if enabled, ok := enabledARGClients[client]; ok && enabled {
+		klog.V(2).Infof("ARG is enabled for %s", client)
```

Comment:
lower log level to v(5)

Reply:
also, it's possible that `enabledARGClients` is nil, then panic?

Reply:
log level updated.
`enabledARGClients` can be nil but I don't think it will panic.

Reply:
would you add a ut where `enabledARGClients` is nil? Thanks.

Reply:
Updated.